app/src/sharedTest/java/org/oppia/app/walkthrough/WalkthroughActivityTest.kt

```diff
@@ -12,14 +13,21 @@ import org.junit.runner.RunWith
 import org.oppia.app.R
 import org.oppia.app.utility.ProgressMatcher.Companion.withProgress
 
-/** Tests for [WalkthroughFragment]. */
+/** Tests for [WalkthroughActivity]. */
 @RunWith(AndroidJUnit4::class)
-class OnboardingFragmentTest {
+class WalkthroughActivityTest {
 
   @Test
-  fun testWalkthroughFragment_checkSkipButton_skipNextButton_progressBar_IsVisible() {
+  fun testWalkthroughFragment_defaultProgress_worksCorrectly() {
     ActivityScenario.launch(WalkthroughActivity::class.java).use {
-      onView(withId(R.id.next_button)).check(matches(isDisplayed()))
+      onView(withId(R.id.walkthrough_progress_bar)).check(matches(withProgress(1)))
+    }
+  }
+
+  @Test
+  fun testWalkthroughFragment_checkFrameLayout_backButton_progressBar_IsVisible() {
+    ActivityScenario.launch(WalkthroughActivity::class.java).use {
```

Comment:
import
` import androidx.test.core.app.ActivityScenario.launch
`
for cleaner code practice i.e.
 `launch(WalkthroughActivity::class.java).use`   instead of  `ActivityScenario.launch(WalkthroughActivity::class.java).use`